docs/io/fits/index.rst

```diff
@@ -724,6 +724,21 @@ the 3rd argument is not a header, it (and other positional arguments) are
 assumed to be the extension specification(s). Header and extension specs can
 also be keyword arguments.
 
+The :func:`diff` function will print a difference report of two fits files,
+including headers and data. The first two arguments must be two fits
+filenames or fits file objects with matching data types (i.e. if using strings
+to specify filenames, both input1 and input2 must be strings).  The third
+argument is an optional extension specification, with the same call format
+of :func:`getheader` and :func:`getdata`.  In addition you can add any keywords
+accepted by the :class:`FITSDiff` class.
+
+::
+
+  >>> diff('inA.fits', 'inB.fits', ext=2)  # get a difference report of extension
```

Comment:
Isn't that already going to happen, there's a `doctest-skip-all` at the top of this file...